WebRoot/css/zm.css

```diff
@@ -1483,6 +1487,22 @@ INPUT[type="text"].search_input-expanded:focus {
 	@AddreddBubbleLeftChild@
 }
 
+.attachmentBubble > :first-child {
+	@AttachmentLeftChild@
```

Comment:
what if some skin developer has changed positioning of child items in attachment container? how this will behave with that?